docs/OMS-Agent-for-Linux.md

```diff
@@ -186,6 +204,57 @@ SHARED_KEY=<Shared Key>
 `sudo /opt/microsoft/omsagent/bin/omsadmin.sh`
 4.	The file will be deleted on successful onboarding
 
+## Onboard a secondary workspace
+From 1.3.0-1, OMSAgent supports to onboard the agent to multiple workspaces.
+Run the omsadmin.sh command supplying the workspace id and key for your workspace, and -m to indicate secondary workspace:
+```
+cd /opt/microsoft/omsagent/bin
+sudo ./omsadmin.sh -w <WorkspaceID> -s <Shared Key> -m <Multi-Homing Marker>
+```
+NOTE: Secondary workspace is currently unable to pull the configuration from OMS service. We are working on it.
+
+## Onboard a secondary workspace using a file
+Reference [Onboarding using a file](#onboarding-using-a-file)
+Add the following line into /etc/omsagent-onboard.conf
+```
+MULTI_HOMING_MARKER=<Any marker you choose>
+```
+
+# Manage Workspaces
+From 1.3.0-1, OMSAgent supports onboarding to multiple workspaces. Here are the commands for workspace management
+
+## List all workspaces
+```
+sudo sh /opt/microsoft/omsagent/bin/omsadmin.sh -l
+```
+
+Sample result for an agent onboarded to 2 workspaces:
+```
+Primary Workspace: 660c7bcd-28d2-453a-84bd-8523e396f600    Success(OMSAgent Registered)
+Workspace: 08d4b0c0-7fac-4159-987c-000271282eff    Success(OMSAgent Registered)

```

Comment:
Have we provided real workspace IDs on GitHub before? I think it might be better to give mocked workspace IDs here, like abcdefghi-1234-5678-9101-jklmnopqrst